pkg/cableengine/ipsec/ipsec.go

```diff
@@ -371,19 +371,19 @@ func (i *Engine) removeCableInternal(cableID string, client *goStrongswanVici.Cl
 	defer i.Unlock()
 
 	/*
-	err = client.Terminate(&goStrongswanVici.TerminateRequest{
-		Child: "submariner-child-" + cableID,
-	})
-	if err != nil {
-		klog.Errorf("Error when terminating child connection %s : %v", cableID, err)
-	}
+		err = client.Terminate(&goStrongswanVici.TerminateRequest{
+			Child: "submariner-child-" + cableID,
+		})
+		if err != nil {
+			klog.Errorf("Error when terminating child connection %s : %v", cableID, err)
+		}
```

Comment:
It looks like this commented code was further indented - was this intended?

Reply:
Unindented.